datahub-web-react/src/app/shared/tags/CreateTagModal.tsx

```diff
@@ -50,7 +50,19 @@ export default function CreateTagModal({ onClose, onBack, visible, tagName, reso
                 })
                     .catch((e) => {
                         message.destroy();
-                        message.error({ content: `Failed to add tag: \n ${e.message || ''}`, duration: 3 });
+                        if (
+                            resources.length > 1 &&
+                            e.message ===
+                                'Unauthorized to perform this action. Please contact your DataHub administrator.'
+                        ) {
+                            message.error({
```

Comment:
It can be more than just datasets - I don't think this specific of an error message will do in this place